pages/api/hakkasan.ts

```diff
@@ -11,18 +12,13 @@ const handler = async (
 ): Promise<void> => {
   const {
     method,
-    query: { ref },
+    query: { ref: refParam },
   } = req;
   switch (method) {
-    case 'POST':
-      break;
-    case 'PUT':
-      break;
-    case 'PATCH':
-      break;
-    default: {
+    case 'GET': {
       try {
         if (process.env.NODE_ENV !== 'production') {
+          delay(2000);
```

Comment:
Looks like you missed an `await` here, otherwise it'll fire off the promise here without waiting for it to resolve.

Reply:
oop :3